docs/alerts/monitors/monitor-faq.md

```diff
@@ -48,6 +48,12 @@ deployment=acme region=us-east1 metric=container_cpu_usage_seconds_total | rate
 
 And so on.
 
+## Is there a limit to the number of monitors I can have?
+
+You can have up to 5,000 monitors (active and inactive) simultaneously.
```

Comment:
```suggestion
See [Sumo Logic Pricing](https://www.sumologic.com/pricing) to learn the number of monitors (active and inactive) you can run simultaneously. You can increase the monitor limit up to 5000 by reaching out to our support team.
```

Reply:
@rmamilla-sumo , thank you for your help! What do you think of these edits?

Reply:
Looks good. Thanks.